manifests/04_operator.yaml

```
@@ -16,7 +16,7 @@ spec:
       serviceAccountName: configure-alertmanager-operator
       containers:
         - name: configure-alertmanager-operator
-          image: quay.io/openshift-sre/configure-alertmanager-operator:v0.1.9-fc79391f
+          image: quay.io/openshift-sre/configure-alertmanager-operator:latest
```

```suggestion
          image: quay.io/app-sre/configure-alertmanager-operator:staging-latest
```

These now live in `app-sre` quay.io organization. I also suggested a change from `latest` to `staging-latest` since there's no longer a `latest` tag.

---

ok, fixed.